src/pages/workspaces/workspace/analysis/modals/AnalysisDuplicator.ts

```diff
@@ -74,19 +74,15 @@ export const AnalysisDuplicator = ({ destroyOld = false, fromLauncher = false, p
           Ajax().Metrics.captureEvent(Events.notebookRename, {
             oldName: printName,
             newName,
-            workspaceName: workspaceInfo.name,
-            workspaceNamespace: workspaceInfo.namespace
+            ...extractWorkspaceDetails(workspaceInfo)
           })
         } else {
           await duplicate()
           // @ts-expect-error
           Ajax().Metrics.captureEvent(Events.notebookCopy, {
```

Comment:
@jdcanas Copying a notebook from a Google workspace to an Azure one fails. Does the IA team already have a ticket for this? If supporting cross-cloud copying is not going to be supported in the near future, I would recommend filtering the workspaces presented in the copy dialog to those in the same cloud platform.

Reply:
Request URL: https://storage.googleapis.com/storage/v1/b/fc-6425fa3a-1f16-4617-b770-bbb509fcbc31/o/notebooks%2FcopyName.ipynb/copyTo/b//o/notebooks%2FcopyNameToAzure.ipynb
Request Method: POST
Status Code: 404 
Remote Address: 172.217.13.240:443
Referrer Policy: strict-origin-when-cross-origin

Reply:
Bringing this back to the team for input.

Reply:
Thank you!